backend/app/views/spree/admin/shared/_image.html.erb

```diff
@@ -1,7 +1,7 @@
 <% size ||= :mini %>
 
-<% if image && image.attachment? %>
-  <%= image_tag image.attachment(size) %>
+<% if image %>
+  <%= image_tag image %>
```

Comment:
This is also missing the `:mini` option.

Reply:
This partial is given an image url (the caller has already decided the size and retrieved the relevant url). I've updated the naming to be clearer.

Does that make sense?